src/main/java/pl/fracz/mcr/comment/AbstractComment.java

```diff
@@ -0,0 +1,52 @@
+package pl.fracz.mcr.comment;
+
+import pl.fracz.mcr.preferences.ApplicationSettings;
+import pl.fracz.mcr.source.SourceFile;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class AbstractComment {
+    private final SimpleDateFormat CREATION_TIME_FORMAT = new SimpleDateFormat("HH:mm dd.MM.yyyy");
+
+    private final Type type;
+
+    private final String author;
+
+    protected SourceFile sourceFile;
+
+    private Date date;
+
+    private int lineNumber;
+
+    public AbstractComment(Type type, int lineNumber) {
+        this.type = type;
```

Comment:
We can have many implementations instead of type: `TextComment`, `VoiceComment`...